server/athenian/api/controllers/goal_controller.py

```diff
@@ -0,0 +1,40 @@
+from operator import attrgetter
+
+from aiohttp import web
+
+from athenian.api.align.goals.templates import TEMPLATES_COLLECTION
+from athenian.api.internal.account import get_user_account_status_from_request
+from athenian.api.models.web import GoalTemplate, NotFoundError
+from athenian.api.request import AthenianWebRequest
+from athenian.api.response import ResponseError, model_response
+
+
+async def get_goal_template(request: AthenianWebRequest, id: int) -> web.Response:
+    """Retrieve a goal template.
+
+    :param id: Numeric identifier of the goal template.
+    :type id: int
+
+    """
+    try:
+        template_def = TEMPLATES_COLLECTION[id]
+    except KeyError:
+        return ResponseError(NotFoundError("Template %d was not found." % id)).response
```

Comment:
Let's add a big TODO to check the account ID <> template ID access.

Reply:
Next PR will add retrieve from DB, and there we check account ownership